Dockerfile.centos

```diff
@@ -61,3 +65,8 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
   && make -j \
   && make install \
   && cd ../ && rm -rf ccache-*
+
+# Install lcov to /usr/local/bin
+# require master to make gcc9 happy
+RUN git clone --depth=1 --branch=master https://github.com/linux-test-project/lcov.git \
```

Comment:
I think we should download `lcov` sources with specified version from their [releases](https://github.com/linux-test-project/lcov/releases) page. Only in this way, the compiled binary of lcov in docker image is same as before.

Reply:
The latest release version is not workable for gcc9.

Reply:
If so , I can specify the commit.